core/src/main/java/org/apache/iceberg/hadoop/HadoopTables.java

```diff
@@ -194,8 +203,24 @@ TableOperations newTableOps(String location) {
     if (location.contains(METADATA_JSON)) {
       return new StaticTableOperations(location, new HadoopFileIO(conf));
     } else {
-      return new HadoopTableOperations(new Path(location), new HadoopFileIO(conf), conf);
+      return new HadoopTableOperations(new Path(location), new HadoopFileIO(conf), conf,
+              createOrGetLockManager(location));
+    }
+  }
+
+  private LockManager createOrGetLockManager(String location) {
+    Map<String, String> properties = new HashMap<>();
+    Iterator<Map.Entry<String, String>> configEntries = this.conf.iterator();
```

Comment:
Nit: no need for the prefix `this.` when accessing a field. We use it when setting a field so that it is easy to see that state is modified, but we don't use it on read.

Reply:
done